src/state.rs

```diff
@@ -265,6 +243,33 @@ impl<'a, S: Storage> Config<'a, S> {
         self.storage
             .set(KEY_CONTRACT_STATUS, &status_u8.to_be_bytes());
     }
+
+    pub fn set_minters(&mut self, minters_to_set: Vec<HumanAddr>) -> StdResult<()> {
+        set_bin_data(&mut self.storage, KEY_MINTERS, &minters_to_set)?;
+
+        Ok(())
+    }
+
+    pub fn add_minters(&mut self, minters_to_add: Vec<HumanAddr>) -> StdResult<()> {
+        let mut minters = self.minters();
+        minters.extend(minters_to_add);
+
+        self.set_minters(minters)
+    }
+
+    pub fn remove_minters(&mut self, minters_to_remove: Vec<HumanAddr>) -> StdResult<()> {
+        let mut minters = self.minters();
+
+        for minter in minters_to_remove {
+            minters.retain(|x| x != &minter);
```

Comment:
This looks better I think @reuvenpo

Reply:
oh, yes, very nice!
`retain` is one of those esoteric methods that nobody knows about, lol